File: Exercicios-Beecrowd/Kotlin/1050.kt
import java.util.*

fun main(args: Array<String>) {

    val entrada= readLine().toString().toInt()
    val dicionario= mapOf(61 to "Brasilia",71 to "Salvador", 11 to "Sao Paulo",21 to "Rio de Janeiro",32 to "Juiz de Fora",19 to "Campinas",27 to "Vitoria", 31 to "Belo Horizonte")
    if (entrada in dicionario.keys){
        println(dicionario[entrada])

    }
    else{
        println("DDD nao cadastrado")
    }

	
}

File: Exercicios-Beecrowd/Python/1050.py
x=int(input())
dicionario={61:'Brasilia',
            71:'Salvador',
            11:'Sao Paulo',
            21:'Rio de Janeiro',
            32:'Juiz de Fora',
            19:'Campinas',
            27:'Vitoria',
            31:'Belo Horizonete'}
if x in dicionario:
    print(dicionario[x])
else:
    print('DDD nao cadastrado')